src/prefect/orion/schemas/core.py

```diff
@@ -90,8 +90,21 @@ class FlowRunPolicy(PrefectBaseModel):
 
     # TODO: Determine how to separate between infrastructure and within-process level
     #       retries
-    max_retries: Optional[int] = None
-    retry_delay_seconds: Optional[float] = None
+    max_retries: int = Field(
+        0,
+        description="The maximum number of retries. Field is not used. Please use `retries` instead.",
+        deprecated=True,
+    )
+    retry_delay_seconds: float = Field(
+        0,
+        description="The delay between retries. Field is not used. Please use `retry_delay` instead.",
+        deprecated=True,
+    )
```

Comment:
I'm not sure of the current status of these fields and how/where they're provided by clients.  Is there any value to having a `@root_validator` that uses the old values to populate the new values (if the new values are `None` and the old values are non-zero)?

Reply:
Hmmm that's a fair point. Would that break any assumptions for old clients @madkinsz?

Reply:
We're here with these changes, right?

Old client -> Sends old fields
New server -> Ignores orchestration behavior

New client -> Sends new fields
Old server -> Throws an error

And introducing a root validator would get us to

Old client -> Sends old fields
New server -> Uses root validator to populate new fields with old
New server -> Expected orchestration behavior is preserved